### PR TITLE
Add Belarusian translations

### PR DESCRIPTION
**Pull request by Zmicier21**
**Summary**

Added Belarusian language

**Translation checklist**

- [x] I have ran the application to make sure my code runs
- [x] I have used the correct [ISO 639-1 code](https://www.alchemysoftware.com/livedocs/ezscript/Topics/Catalyst/Language.htm) for my translations (e.g. `en`)
- [x] I have formatted all changes as described in the [contribution style guide](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/CONTRIBUTING.md#style-guide)
- [x] I have added the language to `supportedLanguages` in alphabetical order
- [x] I have added the language to the table in [README.md](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/README.md#translations) in alphabetical order
- [x] (_If you translated the README_) I have translated the README and linked it in the main README.md
- [x] (_If you translated the README_) I have left the "Translations" section out of the translated README

### CODE REVIEW

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -2096,6 +2163,10 @@ const supportedLanguages = [
     name: 'Brazilian Portuguese (Português Brasileiro)',
     code: 'pt_br',
   },
+  {
+    name: 'Belarusian (Беларуская)',
+    code: 'be',
+  },
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2169]: This need to go above Brazilian Portuguese

File: auto_py_to_exe/web/js/i18n.js
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:1]: I noticed the `ui.button.oneFile` translation is missing

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
There is a [checklist for translations in this template](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/.github/PULL_REQUEST_TEMPLATE/translation.md) - I'll update the PR to have this checklist and you can go through it 👍 

Take a look at #355 if you need an example.

File: auto_py_to_exe/web/js/i18n.js
Comment (Zmicier21) [reply auto_py_to_exe/web/js/i18n.js:1]: As I see it, there is a translation in that place

Comment (brentvollebregt) [reply auto_py_to_exe/web/js/i18n.js:1]: My bad, for some reason it looked like it wasn't there

**Comment by brentvollebregt** [comment]
Could you add an entry to the translations table in the README.md file? Other than that, this looks good.

**Comment by brentvollebregt** [comment]
Hey, I see you have 5 branches in your repo with changes that appear they should be in `Zmicier21:patch-1` - do you need some help getting all these changes into this PR? Happy to move the commits around for you if you need.

**Comment by Zmicier21** [comment]
I don't understand, how use git correctly

File: README.md [hunk 2]
```diff
@@ -132,6 +132,7 @@ If you believe you've found an issue with this tool, please follow the ["Reporti
 | Language                                    | Translator                                                                                   | Translated                           |
 | ------------------------------------------- | -------------------------------------------------------------------------------------------- | ------------------------------------ |
 | Arabic (العربية)                            | [Tayeb-Ali](https://github.com/tayeb-ali)                                                    | UI                                   |
+| Belarusian (Беларуская)                    | [Zmicier21](https://github.com/Zmicier21)            |          UI
```
Comment (brentvollebregt) [inline README.md:135]:
```suggestion
| Belarusian (Беларуская)                     | [Zmicier21](https://github.com/Zmicier21)                                                    | UI                                   |
```

File: README.md
Comment (brentvollebregt) [inline README.md:1]: Could you also add one line under "[Translations of This File](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/README.md?plain=1#L17)" now that you have translated the README 😃

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
Looking great, just two more small things thanks.

> I don't understand, how use git correctly

No worries! I'm happy to help move commits around if you need.

**Review by brentvollebregt — APPROVED** [review]
Just tidied the last couple of things up for you - looks good to go. Thank you for your contribution!

**Comment by brentvollebregt** [comment]
Thanks for the contribution! This has been released in [v2.44.0](https://github.com/brentvollebregt/auto-py-to-exe/releases/tag/v2.44.0) 🚀

**Pull request merged**